### PR TITLE
perf: only render contributors hexagons when in viewport

### PR DESCRIPTION
**Pull request by satazor**
Refs #69.

### CODE REVIEW

File: src/shared/components/community-section/index.js [hunk 1]
```diff
@@ -1,34 +1,55 @@
...
+            <div className={ styles.socialLinksContainer }>
+              <p>{ messages.community.socialNetworksText }</p>
+              <div className={ styles.socialLinks }>
+                <Button translationId="buttonIrcFreenode" path="/test" />
+                <Button translationId="buttonGithub" path="/test" modifier="github" />
+                <Button translationId="buttonTwitter" path="/test" modifier="twitter" />
+              </div>
+            </div>
+          </div>
+          { showContributors ? <iframe frameBorder="0" src={ data.contributorsLink } /> : null }
```
Comment (vasco-santos) [inline src/shared/components/community-section/index.js]: Why not using the `&&` here?

Comment (satazor) [reply src/shared/components/community-section/index.js]: For no reason, will change.

Comment (vasco-santos) [reply src/shared/components/community-section/index.js]: I prefer the `&&` conditional, but it is a minor detail. We can go with the ternary condition as well

**Comment by satazor** [comment]
Done.

**Comment by vasco-santos** [comment]
Will merge this as soon as the CI is green

**Pull request merged**